src/Symfony/Component/Intl/Resources/data/locales/fr_CA.json

```diff
@@ -37,6 +37,7 @@
         "ks": "kashmiri",
         "ks_IN": "kashmiri (Inde)",
         "lu": "luba-katanga",
+        "lu_CD": "luba-katanga (Congo-Kinshasa)",
```

Comment:
now included, because it differs from the parent `fr` (see https://github.com/unicode-org/icu/blob/master/icu4c/source/data/lang/fr.txt#L349 vs. https://github.com/unicode-org/icu/blob/075cefb2e21f57f4cac1bc2868e93dd1b8c077cc/icu4c/source/data/lang/fr_CA.txt#L33)